ladybug/windrose.py

```diff
@@ -222,8 +213,25 @@ def frequency_hours(self, frequency_hours):
         self._frequency_hours = int(frequency_hours)
 
     @property
-    def frequency_intervals(self):
-        return int(math.ceil(self.frequency_maximum / self.frequency_hours))
+    def frequency_intervals_compass(self):
+        """Calculate the number of intervals in the windrose compass."""
```

Comment:
This doesn't seem like the right type of docstring for a property. Property docstrings should start with "Get" or "Get or set" depending on whether they are set-able.